Linux-mac-install.md

```diff
@@ -248,18 +364,18 @@ brew install autoconf automake libtool
 
 ### Step 3. Install the PHP drivers for Microsoft SQL Server
 ```
-sudo pecl install sqlsrv-5.7.1preview
-sudo pecl install pdo_sqlsrv-5.7.1preview
+sudo pecl install sqlsrv
+sudo pecl install pdo_sqlsrv
 ```
 ### Step 4. Install Apache and configure driver loading
 ```
 brew install apache2
 ```
-To find the Apache configuration file for your Apache installation, run 
+To find the Apache configuration file, `httpd.conf`, for your Apache installation, run 
 ```
 apachectl -V | grep SERVER_CONFIG_FILE
 ``` 
-and substitute the path for `httpd.conf` in the following commands:
+The following commands append the required configuration to `httpd.conf`. Be sure to substitute the path returned by the preceding command in place of `/usr/local/etc/httpd/httpd.conf`:
 ```
```

Comment:
Thought we should not subsitute the httpd.conf path?

Reply:
It will likely change though if the user installs Apache via brew, and I don't think earlier versions of macOS (before Catalina) shipped with Apache, so this default path may not be correct for all users.

Reply:
Correction - previous versions did ship with apache. Will modify the apachectl path to get the brew provided executable.